### PR TITLE
Integrate CyberChef-server instance for CyberChef integration

### PR DESCRIPTION
**Pull request by nikstuckenbrock**
## Status
- [x] Ready

## Description
The basic idea was to be able to use CyberChef functionalities in SOAR. However, the [existing integration](https://cortex.marketplace.pan.dev/marketplace/details/CyberChef/) requires a "global" instance of the CyberChef server, which I cannot provide for reasons of separate customer networks and tenants. Hence the idea of integrating the CyberChef server into the Docker image in which the CyberChef integration runs so that I don't have to host the application. This merge request is heavily dependent on PR [#455](https://github.com/demisto/dockerfiles/pull/29725) and must be after it.

### CODE REVIEW

**Comment by samuelFain** [comment]
Hi @nikstuckenbrock, we appreciate your contribution!
Since this PR depends on https://github.com/demisto/dockerfiles/pull/29725, It would make sense to review it after said PR is approved and merged.
So if its ok with you, I'll put a pin on this one until a review is more relevant.
I'm also available on slack / email at: sfainberg@paloaltonetworks.com

**Comment by nikstuckenbrock** [comment]
Hey @samuelFain ,

just a ping, these related PR was merged. :)

File: Packs/CyberChef/ReleaseNotes/1_0_7.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### CyberChef
+
+- Updated the Docker image to: *demisto/cyberchef:1.0.0.104247*.
+- Add cyberchef node package as dependency to execute cyberchef local
```
Comment (samuelFain) [inline Packs/CyberChef/ReleaseNotes/1_0_7.md:7]:
```suggestion
- Added support for cyberchef node package as dependency to execute cyberchef locally.
```
See https://xsoar.pan.dev/docs/documentation/release-notes for more information.

File: Packs/CyberChef/Integrations/CyberChef/CyberChef.yml [hunk 2]
```diff
@@ -3,16 +3,20 @@ commonfields:
   id: CyberChef
   version: -1
 configuration:
+- display: Use the local npm package to execute the bake function
```
Comment (samuelFain) [inline Packs/CyberChef/Integrations/CyberChef/CyberChef.yml:6]: This parameter effects both `cyberchef-bake` and `cyberchef-magic` commands, so why its name only refers to the "bake" functionally? 
Could you also add a short `additionalinfo` to explain when one should use this parameter?

**Review by samuelFain — CHANGES_REQUESTED** [review]
@nikstuckenbrock Looks good! Please see my comments.

Could you please confirm that my understanding is correct? Users who use this version should not experience any backwards compatibility issues and should be able to use the integration as they normally would. These changes will only take effect if the "Use a local npm package to execute the bake function" parameter is checked.

Also, please add the following to `Packs/CyberChef/Integrations/CyberChef/README.md`:
- Add the new `Use the local npm package to execute the bake function` parameter in the _Configure CyberChef on Cortex XSOAR_ section.
- It would be great if you could add a brief explanation of how this parameter is used, or include a link to the official documentation on this topic.

**Review by samuelFain — APPROVED** [review]

**Pull request merged**